app/lib/diff.rb

```diff
@@ -17,8 +17,10 @@ def initialize(old_value, new_value)
 
   def to_html
     @changes.map do |chunk|
-      old_html = CGI.escapeHTML(chunk[:old])
-      new_html = CGI.escapeHTML(chunk[:new])
+      old_ellipsis = chunk[:old].gsub(/((?:\S+\s+){3})(.+)((?:\s+\S+){3})/, '\1...\3')
```

Comment:
It might be a good idea to add a quick description as of what is it that this RegEx does? Not a detailed description, just a short bit giving the next developer a hint as of what it does.